aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -15,75 +15,75 @@ uid: host-and-deploy/iis/modules
+| **Custom Logging**<br>`CustomLoggingModule` | Yes | |
+| **Default Document**<br>`DefaultDocumentModule` | No | [Default Files Middleware](xref:fundamentals/static-files#serve-a-default-document) |
+| **Digest Authentication**<br>`DigestAuthenticationModule` | Yes | |
+| **Directory Browsing**<br>`DirectoryListingModule` | No | [Directory Browsing Middleware](xref:fundamentals/static-files#enable-directory-browsing) |
+| **Dynamic Compression**<br>`DynamicCompressionModule` | Yes | [Response Compression Middleware](xref:performance/response-compression) |
+| **Tracing**<br>`FailedRequestsTracingModule` | Yes | [ASP.NET Core Logging](xref:fundamentals/logging/index#the-tracesource-provider) |
+| **File Caching**<br>`FileCacheModule` | No | [Response Caching Middleware](xref:performance/caching/middleware) |
+| **HTTP Caching**<br>`HttpCacheModule` | No | [Response Caching Middleware](xref:performance/caching/middleware) |
+| **HTTP Logging**<br>`HttpLoggingModule` | Yes | [ASP.NET Core Logging](xref:fundamentals/logging/index)<br>Implementations: [elmah.io](https://github.com/elmahio/Elmah.Io.Extensions.Logging), [Loggr](https://github.com/imobile3/Loggr.Extensions.Logging), [NLog](https://github.com/NLog/NLog.Extensions.Logging), [Serilog](https://github.com/serilog/serilog-extensions-logging)
+| **HTTP Redirection**<br>`HttpRedirectionModule` | Yes | [URL Rewriting Middleware](xref:fundamentals/url-rewriting) |
+| **IIS Client Certificate Mapping Authentication**<br>`IISCertificateMappingAuthenticationModule` | Yes | |
+| **IP and Domain Restrictions**<br>`IpRestrictionModule` | Yes | |
+| **ISAPI Filters**<br>`IsapiFilterModule` | Yes | [Middleware](xref:fundamentals/middleware/index) |
+| **ISAPI**<br>`IsapiModule` | Yes | [Middleware](xref:fundamentals/middleware/index) |
+| **Protocol Support**<br>`ProtocolSupportModule` | Yes | |
+| **Request Filtering**<br>`RequestFilteringModule` | Yes | [URL Rewriting Middleware `IRule`](xref:fundamentals/url-rewriting#irule-based-rule) |
+| **Request Monitor**<br>`RequestMonitorModule` | Yes | |
+| **URL Rewriting**<br>`RewriteModule` | Yes&#8224; | [URL Rewriting Middleware](xref:fundamentals/url-rewriting) |
+| **Server Side Includes**<br>`ServerSideIncludeModule` | No | |
+| **Static Compression**<br>`StaticCompressionModule` | No | [Response Compression Middleware](xref:performance/response-compression) |
+| **Static Content**<br>`StaticFileModule` | No | [Static File Middleware](xref:fundamentals/static-files) |
+| **Token Caching**<br>`TokenCacheModule` | Yes | |
+| **URI Caching**<br>`UriCacheModule` | Yes | |
+| **URL Authorization**<br>`UrlAuthorizationModule` | Yes | [ASP.NET Core Identity](xref:security/authentication/identity) |
+| **Windows Authentication**<br>`WindowsAuthenticationModule` | Yes | |
+
+&#8224;The URL Rewrite Module's `isFile` and `isDirectory` don't work with ASP.NET Core apps due to the changes in [directory structure](xref:host-and-deploy/directory-structure).
```

Comment:
I assume `isFile` and `isDirectory` are properties or attributes. I recommend using one of those words before "don't work" to add clarity.

Reply:
Good idea. They call them "match types" ... e.g. ...

```xml
<add input="{REQUEST_FILENAME}" matchType="isFile" negate="true">
```